partners/gogopool.json

```diff
@@ -36,7 +36,115 @@
             ],
             "badge": {
                 "name": "ggAVAX enjoyer",
-                "description": "Awarded for minting ggAVAX"
+                "description": "Awarded for minting ggAVAX."
```

Comment:
Only one mission is allowed per partner, you can have multiple tasks under the quests